lib/rex/post/meterpreter/ui/console/interactive_channel.rb

```diff
@@ -46,7 +46,7 @@ def _suspend
     # Ask the user if they would like to background the session
     if (prompt_yesno("Background channel #{self.cid}?") == true)
       self.interactive(false)
-
+      Readline::HISTORY.length.times {Readline::HISTORY.pop}
```

Comment:
This also seems like it should be using the history manager's `#pop_context` method instead of explicitly modifying the history itself.

Reply:
I've changed it `pop_context` :+1: